cf_xarray/tests/test_scripts.py

```diff
@@ -1,32 +1,26 @@
 import os
+from tempfile import TemporaryDirectory
```

Comment:
Sorry, this change is not really relevant to this PR but I noticed it and I think this is more robust...